examples/hello-world/step-by-step/cifar10/code/fl/train_model_learner.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torchvision
+from fl.net import Net
+from torchvision import transforms
+
+from nvflare.app_common.abstract.fl_model import FLModel, ParamsType
+from nvflare.app_common.abstract.model_learner import ModelLearner
+from nvflare.app_common.app_constant import ModelName
+
+
+class CIFAR10ModelLearner(ModelLearner):
```

Comment:
should we just call the file name `model_learner.py`? since it does more than training?